lms/services/lti_grading/interface.py

```diff
@@ -1,8 +1,37 @@
 class LTIGradingService:  # pragma: no cover
-    """Service for sending grades back to the LMS."""
+    """
+    Service for sending grades back to the LMS.
 
-    def __init__(self, grading_url):
-        self.grading_url = grading_url
+    From https://www.imsglobal.org/spec/lti-ags/v2p0#nomenclature:
+
+    Line item
+        A line item is usually a column in the tool platform's gradebook; it is able to hold the results associated with a specific activity for a set of users.
+
+    Line item container
+        A line item container has an array of line items.
+
+    Result
+        A result is usually a cell in the tool platform's gradebook; it is unique for a specific line item and user.
+
+    Score
+        A score represents the last score obtained by the student for the tool's activity. It also exposes the current status of the activity (like completed or in progress), and status of the grade.
+
+
+    These are the LTI1.3 concepts but with the exception of line item container they have a counter part in LTI 1.1:
+
+    https://www.imsglobal.org/spec/lti-ags/v2p0#migrating-from-basic-outcomes-service
```

Comment:
```suggestion
    Line item:
        A line item is usually a column in the tool platform's gradebook; it is
        able to hold the results associated with a specific activity 
        (assignment) for a set of users.

    Line item container:
        A line item container has an array of line items (for a course).

    Result:
        A result is usually a cell in the tool platform's gradebook; it is
        unique for a specific line item and user.

    Score:
        A score represents the last score obtained by the student for the
        tool's activity. It also exposes the current status of the activity
        (like completed or in progress), and status of the grade.

    These are the LTI1.3 concepts, but they have a counter-part in LTI 1.1 
    (except for the line item container):

    https://www.imsglobal.org/spec/lti-ags/v2p0#migrating-from-basic-outcomes-service
```

I'll never stop.

This is very good context for an LTI noob such as myself.